chart/templates/proxy_injector.yaml

```diff
@@ -78,6 +78,7 @@ rules:
 - apiGroups: [""]
   resources: ["namespaces"]
   verbs: ["list", "get", "watch"]
+# TODO: this is for creating the webhook config. Can be removed when #2176 lands.
```

Comment:
It actually belongs in both places. The pod listing ability is needed more precisely because of #2559.
I tried using Helm comments (`{{- /* comment */ -}}`) but for some reason that didn't fly. I'll remove anyways these comments and I'll make a note instead :wink: